examples/check-full.yaml

```diff
@@ -106,3 +105,8 @@ jobs:
         with:
           name: ${{ runner.os }}-r${{ matrix.config.r }}-results
           path: check
+
+      - name: Don't use tar from old Rtools to store the cache
+        if: ${{ runner.os == 'Windows' && startsWith(steps.install-r.outputs.installed-r-version, '3.6' ) }}
```

Comment:
Instead of `startsWith()`, this could be `... == '3.6.3'` but I chose this to more obviously match how this R version appears in the build matrix.